src/evaluators/interpolation/PHAL_DOFCellToSide.hpp

```diff
@@ -59,14 +62,66 @@ class DOFCellToSideBase : public PHX::EvaluatorWithBaseImpl<Traits>,
     CELL_VECTOR,
     CELL_TENSOR,
     NODE_SCALAR,
+    NODE_SCALAR_SIDESET,
     NODE_VECTOR,
+    NODE_VECTOR_SIDESET,
     NODE_TENSOR,
-    VERTEX_VECTOR
+    VERTEX_VECTOR,
+    VERTEX_VECTOR_SIDESET
   };
 
   LayoutType layout;
 
   MDFieldMemoizer<Traits> memoizer;
+
+  Kokkos::DynRankView<int, PHX::Device> dimsView;
```

Comment:
Is there a reason to use `DynRankView` over a simple 2d view? I mean, it's always going to be a 2d view, right? Seeing a DynRankView makes me think that we don't know the rank a priori, which confuses me.

Reply:
There are a bunch of existing evaluators where DynRankViews were used for Views with known rank. I was't sure if there was a purpose for doing it like this so I figured I'd just stay consistent with the rest of the code unless someone mentioned otherwise.

Reply:
I think those evaluators are mostly in the BC framework, or anywhere where you need to call Intrepid2 stuff (@mperego can confirm/deny). In those places, it is true that the rank is not known a priori (Intrepid2 works with several layouts, and uses the dyn rank as a shortcut instead of writing several functions, IIRC), so it is "needed".
But if you do know the rank a priori, then there's no need for a dyn rank view (and it might actually be misleading). I'd stick with normal View's if/where possible.

Reply:
Let's try to get rid of this completely and use a static array. It looks like 5 is the max size?

Reply:
It's a 2d view of size (numSides, numNodes). I assume the maximum number of sides is 6 (prism/wedge?), and the maximum number of nodes on a side is 4?

Reply:
this one is a 1D view of dims and it looks like dims[4] is the largest value used in the kernel

Reply:
Yeah, I was about to edit, I thought this was in reference to sideNodes. I will make this one a static array of size 5.